manifest-pcf.yml

```diff
@@ -8,16 +8,17 @@ applications:
   - fortunes-db
   - config-server
   - service-registry
-  #env:
-    # Replace with API URI of target PCF environment
-    #CF_TARGET: https://api.yourpcfenvironment.local
+  env:
+    CF_TARGET: https://api.cf.gold.springapps.io
```

Comment:
Do we want to leave the RGB environment here?

Reply:
Nope! good catch :)